app/src/ui/octicons/icon-preview-dialog.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react'
+import { Dialog, DialogContent } from '../dialog'
+import * as octicons from './octicons.generated'
+import { Octicon, OcticonSymbolVariant, OcticonSymbolVariants } from '.'
+
+interface IIconPreviewDialogProps {
+  readonly onDismissed: () => void
+}
+
+export class IconPreviewDialog extends React.Component<
+  IIconPreviewDialogProps,
+  {}
+> {
+  public render() {
```

Comment:
```suggestion
export class IconPreviewDialog extends React.Component<IIconPreviewDialogProps> {
  public render() {
```

Nit: The second parameter is not required